src/entities/EmptyCell.cpp

```diff
@@ -22,3 +22,19 @@
  * @package entities
  * @author Jean LELIEVRE <Jean.LELIEVRE@supinfo.com>
  */
+
+#include "EmptyCell.hpp"
+
+using namespace entities;
+
+const std::string EmptyCell::PATH_EMPTY_CELL = "res/cells/empty.png";
+
+/**
+ *
+ */
+EmptyCell::EmptyCell() : Cell()
```

Comment:
check if it is possible to directly specify the path in Cell constructor parameters ( send PATH_EMPTY_CELL ), like that, it is possible to refactor the two next lines